operator/src/main/resources/application.properties

```diff
@@ -0,0 +1,20 @@
+console.selector=
+
+quarkus.container-image.build=true
+#quarkus.container-image.group=
+quarkus.container-image.name=console-operator
+
+quarkus.operator-sdk.activate-leader-election-for-profiles=prod
+quarkus.operator-sdk.controllers."consolereconciler".selector=${console.selector}
```

Comment:
why the quotes?  hang over from the old name?

Reply:
Usually, whenever there is some application-provided element of a property, it is quoted in the full property key. It's likely because the name of the reconciler could have a dot character itself and this clarifies that the quoted element is a variable section. This is the case for log configuration also, where the FQCN (the log category) is a single element of the property key, but itself contains dots.